### DISCUSSION TITLE
How to config a other language in configuration.yaml file

### DISCUSSION THREAD

**Discussion by ThoEngel** [astrandb/miele#148] [Q&A] [answered]
How to config a other language in configuration.yaml file?

**Comment by astrandb** [comment, marked as answer]
The language entry in configuration.yaml is not used in this integration. It is still allowed for compatibility with other similar integrations.
You can change language in your personal profile. However, device names and entity names are always in english. This is due to limitations in Home Assistant.
I am putting this to the to-do list to handle legacy configuration in the background.
